### DISCUSSION TITLE
Question about DateSystem.FindDayBetween

### DISCUSSION THREAD

**Discussion by askelia** [nager/Nager.Date#418] [General]
Hi,
I'm surprised by the last return (`return startDay;`) in DateSystem.FindDayBetween.
I think it should return null rather than startDay, shouldn't it ? 
If i do `FindDayBetween(2022,08,25, 2022,08,28, DayOfWeek.Tuesday)`, there's no tuesday between those 2 dates so I should get null, not 2022-08-25.
It seems to be used only in FinlandProvider with a 7 days interval so it shouldn't cause any problem.

What do you think ?

**Comment by tinohager** [comment]
Hi this was a bug, i have fixed it
https://github.com/nager/Nager.Date/commit/9c0ad04a0a24bf78c4b9bddafa250a4ed2bc36a9
